### PR TITLE
Applied prettier 

### PR DESCRIPTION
**Pull request by unnamedsunshine**
image

### CODE REVIEW

**Comment by jameswinegar** [comment]
Is there anyway we can apply prettier automatically? We should standardize our formatting instead of doing one offs.

File: backend/src/controller/author.ts [hunk 1]
```diff
@@ -2,103 +2,101 @@ import NotFoundError from "../errors/errors";
...
-    const authorPayload:AuthorPayload = {
-        name:req.body.name, 
-        bio:req.body.bio
-    };
-    if (!authorPayload.name || !authorPayload.bio){
-        res.status(400).send("Bad request. Missing required field(s).");
-        return;
-    }
-    try{
-        const createdAuthor = await authorService(dbConnection).create(authorPayload);
```
Comment (ideheer) [inline backend/src/controller/author.ts:21]: Here's a comment

File: backend/src/controller/author.ts [hunk 2]
```diff
@@ -2,103 +2,101 @@ import NotFoundError from "../errors/errors";
...
 
 //Get all authors
-const getAllAuthors = async (req:Request, res:Response) => {
-    try{
-        const authorList = await authorService(dbConnection).getAll();
-        res.json(authorList);
-    }
-    catch(error:any){
-        res.status(500).send(error.message);
-    }
```
Comment (ideheer) [inline backend/src/controller/author.ts:37]: This is a review starting comment.

**Review by ideheer — APPROVED** [review]
nit - maybe use tabs instead of spaces
lgtm
LGTM

**Pull request merged**